backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -128,32 +67,47 @@ private static Message createMessageWithEmail(MimeMessage emailContent)
    * @return the MimeMessage to be used to send email
    * @throws MessagingException if there was a problem accessing the Store
    */
-  private MimeMessage createEmail(String subject, String bodyText) throws MessagingException {
+  private MimeMessage createEmail(String recipientUsername, String subject, String bodyText)
+      throws MessagingException {
 
     Properties props = new Properties();
     Session session = Session.getDefaultInstance(props, /* authenticator= */ null);
 
     MimeMessage email = new MimeMessage(session);
 
     email.setFrom(new InternetAddress(APPLICATION_EMAIL));
-    email.addRecipient(RecipientType.TO, new InternetAddress(recipientEmail));
+    email.addRecipient(RecipientType.TO, new InternetAddress(recipientUsername + "@google.com"));
     email.setSubject(subject);
     email.setText(bodyText);
     return email;
   }
 
   /** Function that access its api and using it sends an email */
-  //   TODO(#35): Create a dummy email for ad lib itself to send emails.
   //   TODO(#36): Replace body to send real link to user instead of generic.
-  public void notifyUser() throws MessagingException, IOException, GeneralSecurityException {
-    NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+  public void sendMatchEmail(String firstMatchRecipientName)
+      throws MessagingException, IOException {
+    MimeMessage email =
+        createEmail(
+            firstMatchRecipientName,
+            "Ad-lib: We found you a match !",
+            " We found you a match with matchUsername. "
```

Comment:
Is there a way we can get the email address/username from servlet Users API or even the Match object here to replace the "matchUsername" with an actual username instead?

Reply:
I have changed it to use the string name parameter that has been passed into the function.